dashboard/module.properties

```diff
@@ -0,0 +1,8 @@
+Name: PanoramaWeb Dashboard
+ModuleClass: org.labkey.dashboard.DashboardModule
+ModuleDependencies: TargetedMS
```

Comment:
Dependencies should now be declared in the module's build.gradle. See the TargetedMS module's copy for an example